### PR TITLE
Python - The `is` Operator

### PR DESCRIPTION
**Pull request by kapnobatai137**
- move the is operator insight in the Memory Allocation workout 

TODOs
- [ ] resolve comments after merging

### CODE REVIEW

**Comment by enki-hq** [comment]
### Topic

| Topic Name | Error Message |
|:-:|:-:|
| error-not-found | Topic should have at least one workout with the "workout" aspect. |
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Topic

| Topic Name | Error Message |
|:-:|:-:|
| error-not-found | Topic should have at least one workout with the "workout" aspect. |

> If you notice anything wrong with these rules contact andrei@enki.com.

File: python/core/memory-allocation/README.md [hunk 1]
```diff
@@ -1,12 +1,15 @@
 name: Memory Allocation
+
 description: On how memory allocation and de-allocation works.
+
 insights:
+  - the-is-operator
```
Comment (Loopiezlol) [inline python/core/memory-allocation/README.md:6]: lgtm, but wouldn't it be better if we first introduce pointers and references then use them as explanation for this insight?

**Review by Loopiezlol — APPROVED** [review]

File: python/core/memory-allocation/README.md (hunk 1 above)
Comment (kapnobatai137) [reply python/core/memory-allocation/README.md:6]: yes, that would be better :+1:

**Comment by enki-hq** [comment]
### Topic

| Topic Name | Error Message |
|:-:|:-:|
| error-not-found | Topic should have at least one workout with the "workout" aspect. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Pull request merged**